### ISSUE TITLE
Staging Review finding: Success message not announced to screen reader users

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#91820]
## Value Proposition

As a veteran who uses a screen reader to navigate CST, I want my screen reader to read the success message when I successfully submit a waiver, so that I know my action was completed successfully.

_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Benefits Management Tools
**Product:** Claim Status Tool
**Feature:** Improved 5103 requests and waiver experience

## Findings details
**VA.gov Experience Standard - issue:** [User encounters content that doesn't match other content on VA.gov.](https://design.va.gov/about/experience-standards/#user-encounters-content-that-is-similar-to-other-content-on-vagov)
**VA.gov Experience Standard - category:** [Consistency](https://design.va.gov/about/experience-standards/#category-consistency)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

From the 5103 Evidence Notice page when you submit your evidence waiver, you're taken back to the Status page with a success alert at the top of the page. However, that message is not immediately announced to screen reader users when it appears.

Screen reader users should be able to recognize from the breadcrumbs that are announced that they've moved to a new page. But they won't immediately know if what they did worked without having to explore the page.

<details><summary>Link, screenshot or steps to recreate</summary>

Steps to reproduce:
1. Log in with test user Kyle Cole.
2. Turn on a screen reader (easiest for testing: VoiceOver on Mac or NVDA on Windows).
3. Tab to the checkbox and check it.
4. Tab to "Submit evidence waiver" button.
5. Press [enter].
6. Observe that the breadcrumbs are the announced but no status update about the submission is announced.

</details>

## Recommended action

If possible, send focus to the success alert.

# Engineering Notes/ Tasks
- use the feature flag `cst_5103_update_enabled` and when enabled we should change focus of the page to the heading in the success alert.

## References

- **Accessibility Defect Severity:** [3: Medium. Should be fixed in 1-3 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-3)
- **WCAG Success Criteria:** N/A
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

## Acceptance Criteria

- [ ] Behind 5103 feature flag
- [ ] Users using a screen reader can hear the success message after submitting a 5103 waiver
- [ ] The focus goes to the success alert heading instead of to the breadcrumbs on the page

**Comment by davidakennedy** [comment]
> If possible, send focus to the success alert.

Note for our engineers: We want to move focus to the heading in the alert, not the alert container.

**Comment by pmclaren19** [comment]
this was reviewed in eng refinement

**Comment by pmclaren19** [comment]
Tested with DK using a screen reader and it works correctly
